src/components/common/input/checkbox.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
```

Comment:
We have a checkbox in axis here: https://axis.centrifuge.io/?path=/story/theme-all-components--light-mode (github.com/centrifuge/axis) are you able to use this one instead?

Reply:
Yes, I have changed the CheckBox. But for Search Input component the styles seems different with smaller fonts.

Reply:
While testing i found errors are not props of the Axis Checkbox. So I have switch back the Checkbox.